libPhoneNumber-GeocodingParser/libPhoneNumber-GeocodingParser/SQLiteDatabaseConnection.h

```diff
@@ -0,0 +1,24 @@
+//
+//  SQLiteDatabaseConnection.h
+//  libPhoneNumber-GeocodingParser
+//
+//  Created by Rastaar Haghi on 7/1/20.
+//  Copyright © 2020 Google LLC. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <sqlite3.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SQLiteDatabaseConnection : NSObject
+
+- (instancetype)initWithCountryCode:(NSString *)countryCode
+                       withLanguage:(NSString *)language;
```

Comment:
I think you can omit the "with" here. `language` is enough.

Reply:
noted.